db/schema.rb

```diff
@@ -2526,6 +2526,38 @@
             ORDER BY alert_generation_runs.school_id, alert_generation_runs.created_at DESC) latest_runs
     WHERE ((benefit_estimate.alert_generation_run_id = latest_runs.id) AND (additional.alert_generation_run_id = latest_runs.id));
   SQL
+  create_view "recent_change_in_baseloads", sql_definition: <<-SQL
```

Comment:
schema.rb is all messed up in terms of ordering again - have left as is from my local machine - don't know if to try and "fix" it though or just merge this in?